docs/examples/cart-service/README.md

```diff
@@ -111,7 +116,7 @@ export class Cart {
       // In this case, we just add the new item.
       return new Cart(
         currentCart.id,
-        Cart.newItems(currentCart.cartItems, event.sku, event.quantity)
+        Cart.newItems(currentCart.items, event.sku, event.quantity)
```

Comment:
@tainguyenbui Could you please also change the declaration of the field? (in the Cart constructor, right now it is called `cartItems` too)

Reply:
updated @alvaroloes 7e9407a959bd90c01c78d2044f83a96cab743da4 thanks!